### DISCUSSION TITLE
Is there any way to change 'gauge_reports_dir' run time in before_suite hook?

### DISCUSSION THREAD

**Discussion by samuelbenison** [getgauge/gauge#1924] [General]
Hi,

I am using Gauge in Python. I have a small question about the reports directory that stores the HTML report file.

I am trying to replace the 'gauge_reports_dir' value provided in env properties file with a new path generated at run time.

I tried to set the value of the variable `os.environ[variable_name] =/new/reports/path` in before_suite hook but it didn't work?

Any idea how I change the reports directory in runtime? Please advise.

Thanks

**Comment by sriv** [comment]
You'll need to set it before launching gauge either in properties file or as an env variable. There isn't a way to alter these properties once gauge is launched.

**Reply by samuelbenison** [reply]
That's what I thought. Thanks. :)
